### ISSUE TITLE
conversionMap.txt doesn't match with project

### ISSUE THREAD

**Issue by nguyenthuyan** [rockbruno/swiftshield#96]
I checked conversionMap.txt after running automatic mode. i found some bugs: 
1. For example, in conversionMap file, it has:
 sceneWillResignActive ===> ceeczwfOBgjINyuheRwFOuPLysIwrDZg 
but in the project, it's still: sceneWillResignActive

2. i wonder is variable in project will or will not be obfuscated? in my project, no variable is obfuscated. i'm using 3.5.1 SwiftShield.
Thanks

**Comment by rockbruno** [comment]
`sceneWillResignActive` is ignored because that's an UIKit method. It's a mistake that these properties end up in the final map (happens because the decision to ignore occurs later in the execution)

**Comment by nguyenthuyan** [comment]
thanks, 
and how about variable  in function? in my project, it's not be obfuscated. is this right ? . eg:
    private func qVHDwcBqpQGPinepJOUhnTiJcJJvXWEc() -> String {
        let testsample = "t"
        let testsample1 = "t1"
        let testsample2 = testsample + testsample1
        return testsample2
    }

**Comment by rockbruno** [comment]
Correct, we can't identify them for now. https://github.com/rockbruno/swiftshield/blob/master/SOURCEKITISSUES.md
